tests/unit/states/test_win_lgpo.py

```diff
@@ -174,6 +163,19 @@ def setUp(self):
         with patch.dict(win_lgpo.__opts__, {'test': False}):
             win_lgpo.set_(name='nc_state', computer_policy=computer_policy)
 
+    def _convert_to_unicode(self, data):
```

Comment:
looks like this same code is being used 3 times. anyway way we can condense this to one function that we call?

Reply:
Used the function in the state